applovin_max/lib/src/max_ad_view.dart

```diff
@@ -73,7 +73,10 @@ class MaxAdView extends StatefulWidget {
   final AdViewAdListener? listener;
 
   /// Whether auto-refresh is enabled. Defaults to `true`.
-  final bool isAutoRefreshEnabled;
+  final bool autoRefreshEnabled;
+
+  /// Whether adaptive banner sizing is enabled. Defaults to `true`.
+  final bool adaptiveBannerEnabled;
```

Comment:
<img width="869" alt="image" src="https://github.com/user-attachments/assets/859e300f-a439-465d-b802-0f0d31423cbd" />

Reply:
Ok, let me make sure that we should have these for both React Native and Flutter?

`isAutoRefreshEnabled`
`isAdaptiveBannerEnabled`

Reply:
Yes, it's more-or-less a universal standard.